packages/db/src/lib/granule.ts

```diff
@@ -354,3 +357,39 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+ * @param {Array<ApiGranule>} [granules] - updated ApiGranule records
+ * @returns {Promise<void>}
+ */
+export const updateGranuleAndFiles = async (
+  knexOrTransaction: Knex | Knex.Transaction,
+  granules: Array<ApiGranule>
+):Promise<void> => {
+  const {
+    granules: granulesTable,
+    files: filesTable,
+  } = TableNames;
+  await Promise.all(granules.map(async (granule) => {
+    const pgGranule = await translateApiGranuleToPostgresGranule({
```

Comment:
Theory: Should the DB package be taking API granules as input for anything other than translation generally?      I see we're doing it for *fields* in other methods, so I'm not resting on package dogma, but seeing 'take a set of API granules and translate' in the method makes me wonder if we should be offloading that concurrency to the calling method and just take a set of updates to incoming Knex objects.